.github/scripts/shellCheck.sh

```diff
@@ -1,48 +1,49 @@
 #!/bin/bash
 
-CURRENT_DIR=$(pwd)
-ROOT_DIR="$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")"
-
-cd "$ROOT_DIR" || exit 1
-
-source ./.github/scripts/shellUtils.sh
-
-declare -r DIRECTORIES_TO_IGNORE=(
-  './node_modules'
-  './vendor'
-  './ios/Pods'
-  './.husky'
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+readonly SCRIPT_DIR
+ROOT_DIR="$(dirname "$(dirname "$SCRIPT_DIR")")"
+readonly ROOT_DIR
+
+source "$SCRIPT_DIR"/shellUtils.sh
+
+readonly DIRECTORIES_TO_IGNORE=(
+    "$ROOT_DIR/node_modules"
+    "$ROOT_DIR/vendor"
+    "$ROOT_DIR/ios/Pods"
+    "$ROOT_DIR/.husky"
 )
 
 # This lists all shell scripts in this repo except those in directories we want to ignore
 read -ra IGNORE_DIRS < <(join_by_string ' -o -path ' "${DIRECTORIES_TO_IGNORE[@]}")
-SHELL_SCRIPTS=$(find . -type d \( -path "${IGNORE_DIRS[@]}" \) -prune -o -name '*.sh' -print)
-info "👀 Linting the following shell scripts using ShellCheck: $SHELL_SCRIPTS"
-info
+SHELL_SCRIPTS=$(find "$ROOT_DIR" -type d \( -path "${IGNORE_DIRS[@]}" \) -prune -o -name '*.sh' -print)
```

Comment:
Always quote for safety

```suggestion
SHELL_SCRIPTS="$(find "$ROOT_DIR" -type d \( -path "${IGNORE_DIRS[@]}" \) -prune -o -name '*.sh' -print)"
```